### PR TITLE
Table - clickable row

### LINKED ISSUE #477: Table - clickable row
Table row should be clickable

See https://github.com/CraveFood/webapp/issues/2981

### PR DESCRIPTION
**Pull request by alcferreira**
## Closes Issues

* Closes #477

### Reviewer 1

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### Reviewer 2

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### CODE REVIEW

File: packages/table/src/Table.js [hunk 1]
```diff
@@ -329,6 +389,7 @@ class Table extends React.Component {
     rowGroupKey: PropTypes.string,
     flatGroupCondition: PropTypes.func,
     onTitleClick: PropTypes.func,
+    onRowClick: PropTypes.func,
```
Comment (fczuardi) [inline packages/table/src/Table.js]: please add this property to the README

File: packages/input-checkbox/package.json [hunk 2]
```diff
@@ -24,21 +21,15 @@
...
-  ],
+  "keywords": ["farmblocks", "react", "widget", "checkbox", "form", "input"],
   "peerDependencies": {
     "prop-types": "^15.6.0",
     "react": "^16.0.0",
     "styled-components": "^3.0.2"
   },
   "dependencies": {
     "@crave/farmblocks-hoc-disabled-tooltip": "^1.1.9",
+    "@crave/farmblocks-input-checkbox": "^2.0.0",
```
Comment (fczuardi) [inline packages/input-checkbox/package.json]: is this correct? `farmblocks-input-checkbox` having `farmblocks-input-checkbox` as a dependency?

**Comment by vNakamura** [comment]
Links/Buttons will trigger both it's own `onClick` and `onRowClick`:
```javascript
.add(
  "With row click listener and link",
  withInfo()(() => (
    <Table data={fruits} onRowClick={action("onRowClick")}>
      <Column
        title="Fruit"
        customCell={row => (
          <Link onClick={action("onLinkClick")}>{row.name}</Link>
        )}
      />
      <Column title="Price" text={row => row.price} />
    </Table>
  ))
)
```


I'm not sure if there's a good way to detect that a child component has `onClick`. Maybe we could simply add instructions in the docs to use `event.stopPorpagation()` in those cases.

File: packages/input-checkbox/package.json (hunk 2 above)
Comment (alcferreira) [reply packages/input-checkbox/package.json]: Nope... I installed it on the wrong package and I thought I had removed... Will do that, thanks

**Comment by alcferreira** [comment]
@vNakamura Yep... stopPropagation is the way to avoid that behaviour... I'll add this note to docs

**Pull request merged**